src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. Here, you're creating a new instance of `ColorSupplier` every time `getRandomBall` is called. It's better to have a single instance as a class-level field and use it to get the random color.